py_ocsf_models/objects/image.py

```diff
@@ -17,7 +17,7 @@ class Image(BaseModel):
     """
 
     tag: Optional[str]
-    labels: Optional[list[str]]
+    labels: Optional[dict]
```

Comment:
This will not comply with the OCSF specification https://schema.ocsf.io/1.2.0/objects/image?extensions=. `labels` must be a String Array.